Magento2/Tests/Methods/DeprecatedModelMethodUnitTest.inc

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$model->getResource()->save($model);
+
+$model->getResource()->load($model, $id);
+
+$model->getResource()->delete($model);
```

Comment:
Please add here something like.

```
$model->anotherMethodWithResource()->save($model);
```

We need have cases here to make sure check does not cause false positive findings.